src/main/java/mate/academy/Main.java

```diff
@@ -2,11 +2,14 @@
 
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
 
 public class Main {
+    private static final Integer THREAD_POOL_SIZE = 4;
```

Comment:
The `THREAD_POOL_SIZE` constant is using wrapper class `Integer` instead of the primitive `int` type. Since the size of a thread pool is not going to be `null`, it's better to use the primitive type here to avoid unnecessary boxing and unboxing.